src/main/java/uk/gov/ons/census/casesvc/service/CcsToFieldService.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.ons.census.casesvc.service;
+
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.census.casesvc.model.dto.CcsToFwmt;
+import uk.gov.ons.census.casesvc.model.entity.Case;
+
+@Component
+public class CcsToFieldService {
+
+  private static final String ACTION_FIELD_BINDING = "Action.Field.binding";
```

Comment:
we put bindings into the `application.yml`

Reply:
done

Reply:
done